src/main/java/friday/command/ExitCommand.java

```diff
@@ -0,0 +1,28 @@
+package friday.command;
+
+import friday.fridayexceptions.FridayException;
+import friday.storage.Storage;
+import friday.tasklist.TaskList;
+import friday.ui.Ui;
+
+import java.io.IOException;
+
+public class ExitCommand extends Command {
+    public ExitCommand(String fullCommand) {
+        super(fullCommand);
+    }
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws FridayException {
+        try {
+            //save allTasks into TaskList file
+            Storage.saveFile(TaskList.returnList());
+            ui.farewell();
+        } catch (IOException e){
```

Comment:
```suggestion
        } catch (IOException e) {
```
Minor coding standard violation, as mentioned earlier you can set up a checkstyle so you don't need to comb through everything yourself.